en/docs/install-and-setup/deploy-with-docker.md

```
@@ -89,7 +89,7 @@ This step builds container image for WSO2 API Manager with WSO2 Open Banking API
 1. Navigate to `<OB_DOCKER_HOME>/samples/wso2am_with_obam` directory.
 2. Execute `docker build` command as shown below.
 
-     ```tab='Sample'
+    ```tab='Sample'
```

markdownlint-cli2 (0.20.0)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)